scripts/gen-sample-config.js

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2019 New Vector Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The sample config is generated as follows:
+// 1. Start with SdkConfig's DEFAULTS
+// 2. Overwrite with features and Settings defaults
+// 3. Overwrite with samples defined here
+
+const SdkConfig = require("matrix-react-sdk/lib/SdkConfig");
+const fs = require("fs");
+
+const SAMPLES = {
+    default_server_name: "matrix.org",
```

Comment:
I haven't decided yet if this sample info is better here or inline in `Settings.js` with each setting... I just worry it might get forgotten over here, but I suppose the idea is you'd run this script whenever a new setting is added, so hopefully not forgotten then...

Reply:
The problem is also that the config is more than just settings. Personally, I'd rather us fill out the defaults for SdkConfig so we don't need this here, but that raises problems. The first problem being that `default_server_name` *or* `default_hs_url` (and is_url) can be defined - it's hard to represent that in a config.

Reply:
Hi, one of my client needs private instant Messaging software like whatsapp. is it possible to develop riot for a private company. Please mail me, if anyone can solve my problem. Email: haydenj951@gmail.com ll skype: almamun.dev ll Whatsapp: +8801723894682